common/api-review/performance-exp.api.md

```diff
@@ -0,0 +1,24 @@
+## API Report File for "@firebase/performance-exp"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { FirebaseApp } from '@firebase/app-types-exp';
+import { FirebasePerformance } from '@firebase/performance-types-exp';
+import { PerformanceSettings } from '@firebase/performance-types-exp';
+import { PerformanceTrace } from '@firebase/performance-types-exp';
+
+// @public (undocumented)
+export function getPerformance(app: FirebaseApp, settings?: PerformanceSettings): FirebasePerformance;
+
+// @public (undocumented)
+export function registerPerformance(): void;

```

Comment:
@jposuna It is the auto-generated API review file that should be checked in and be reviewed as part of PRs that touches the performance SDK.

We need to make a few changes here:
- Do not export `registerPerformance()`.
- Add comments for the APIs that are noted as `undocumented`
- Add release tag to all exported things. For perf, all of them should be `@public`. See [this](https://github.com/firebase/firebase-js-sdk/blob/master/packages-exp/installations-exp/src/api/get-id.ts#L27) as an example.

Reply:
Done.